app/components/root/root_container.js

```diff
@@ -6,14 +6,17 @@ import {connect} from 'react-redux';
 
 import Config from 'assets/config.json';
 
-import {loadConfigAndLicense} from 'app/actions/views/root';
-import {setAppState} from 'service/actions/general';
+import {flushToStorage} from 'app/actions/storage';
+import {goToNotification, loadConfigAndLicense, queueNotification} from 'app/actions/views/root';
+import {setAppState, setDeviceToken} from 'service/actions/general';
 
 import Root from './root';
 
 function mapStateToProps(state, ownProps) {
     const users = state.entities.users;
     const currentUserId = users.currentId;
+    const {currentId: currentTeamId} = state.entities.teams;
```

Comment:
Similar to @thomchop's comment about the requests earlier this week, perhaps we should consider renaming `users.currentId`, `teams.currentId`, and `users.currentId` so we don't have to alias them